contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -369,6 +372,12 @@ def write_assessment_item(assessment_item, zf):
     result = render_to_string(template, context).encode('utf-8', "ignore")
     write_to_zipfile("{0}.json".format(assessment_item.assessment_id), result, zf)
 
+def process_formulas(content):
+    for match in re.finditer(ur'\$(\$[^\$]+\$)\$', content):
```

Comment:
I know we did previously fix some issues with exercises that had dollar amounts in them, yeah, so we need to watch for that